server/storage/types.go

```diff
@@ -16,12 +16,16 @@ type (
 	ClusterState int
 	ShardRole    int
 	NodeState    int
+	TopologyType string
 )
 
 const (
 	ClusterStateEmpty ClusterState = iota + 1
 	ClusterStateStable
 	ClusterStatePrepare
+
+	TopologyTypeStatic  = "static"
```

Comment:
Use enum?

Reply:
`TopologyTypeStatic` need to be string, so `iota` is not used directly here.